tests/tacacs/test_accounting.py

```diff
@@ -122,6 +123,21 @@ def check_local_no_other_user_log(duthost, tacacs_creds):
     pytest_assert(len(logs) == 0, "Expected to find no accounting logs but found: {}".format(logs))
 
 
+def check_server_received(ptfhost, data):
+    """
+        Check if tacacs server received the data.
+    """
+    hex = binascii.hexlify(data.encode('ascii'))
+    hex_string = hex.decode()
+
+    # Extract received data from tac_plus.log, then use grep to check if the received data contains hex_string
+    sed_command = "sed -n 's/.*-> 0x\(..\).*/\\1/p'  /var/log/tac_plus.log | sed ':a; N; $!ba; s/\\n//g' | grep '{0}'".format(hex_string) # noqa W605 E501
```

Comment:
>W605 [](http://example.com/codeflow?start=145&length=4)

I guess you intend to suppress some static code analysis warnings or errors. Could you explain in details so we can review in PR?

Reply:
Fixed, add detail in comments:


            W605 : Invalid escape sequence. Flake8 can't handle sed command escape sequence, so will report false alert.
            E501 : Line too long. Following sed command difficult to split to multiple line.